lib/application.js

```diff
@@ -234,16 +235,23 @@ class Application extends api.events.EventEmitter {
 
   loadPlaces(callback) {
     api.metasync.each(PLACES, (placeName, cb) => {
-      this.loadPlaceScripts(placeName, cb);
+      if (placeName === 'static') {
+        cb();
+        return;
+      } else if (placeName === 'schemas') {
+        const dir = api.path.join(this.dir, 'schemas');
+        api.metaschema.fs.loadAndCreate(dir, null, (error, schemas) => {
```

Comment:
Shouldn't we at least pass `api` as a second argument to the `loadAndCreate`?

Reply:
@tshemsedinov we cannot pass `null`, we need at least `api.common.Uint64` and `gs` available in schemas.